java/ql/test/library-tests/errortype/Test.java

```diff
@@ -7,6 +7,16 @@ public NoSuchClass test() {
     return nsc;
   }
 
+  static class GenericClass<T> {
+    public void method() { }
+  }
+
+  public testDispatch() {
```

Comment:
```suggestion
  public void testDispatch() {
```
Unless your intent is to test broken syntax as well :)

Reply:
Thanks!